misc/FFNx.blit.frag

```diff
@@ -0,0 +1,27 @@
+/****************************************************************************/
+//    Copyright (C) 2022 Cosmos                                             //
+//                                                                          //
+//    This file is part of FFNx                                             //
+//                                                                          //
+//    FFNx is free software: you can redistribute it and/or modify          //
+//    it under the terms of the GNU General Public License as published by  //
+//    the Free Software Foundation, either version 3 of the License         //
+//                                                                          //
+//    FFNx is distributed in the hope that it will be useful,               //
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of        //
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         //
+//    GNU General Public License for more details.                          //
+/****************************************************************************/
+
+$input v_texcoord0
+
+#include <bgfx/bgfx_shader.sh>
+#include "FFNx.common.sh"
+
+SAMPLER2D(tex_0, 0);
+
+void main()
+{
+	vec4 color = texture2D(tex_0, v_texcoord0.xy);
+	gl_FragColor = color;
```

Comment:
You can optimize this by simply doing
```glsl
gl_FragColor = texture2D(tex_0, v_texcoord0.xy);
```
GPU drivers will be much happier too.

Reply:
Done! GPU drivers happy now :)